.yarn/patches/@lavamoat-lavapack-npm-3.3.0-5a61b5374d.patch

```diff
@@ -0,0 +1,55 @@
+diff --git a/src/runtime.js b/src/runtime.js
+index 97c9ddb9b76a2fe31cf9b202cb491f15232232ff..02d657e84d6d4572d9b6eb5e73caa01380c89b0c 100644
+--- a/src/runtime.js
++++ b/src/runtime.js
+@@ -10831,6 +10831,10 @@ function makeGetEndowmentsForConfig ({ createFunctionWrapper }) {
+ function getPropertyDescriptorDeep (target, key) {
+   let receiver = target
+   while (true) {
++    // abort if this is the end of the prototype chain.
++    if (!receiver) {
++      return { prop: null, receiver: null }
++    }
```

Comment:
Can you share a case for which this was necessary? Looks like a general fix.

Reply:
This was throwing when running in a sandboxed iframe where `document.body.appendChild` was null. Upstreamed fix: https://github.com/LavaMoat/LavaMoat/pull/463

Reply:
ok, let's deal with it there.